snooty/parser.py

```diff
@@ -1527,7 +1527,7 @@ def __init__(
         self.expensive_operation_cache: Cache[FileId] = Cache()
         self.backend.on_config(self.config, branch)
 
-    def get_page_ast(self, path: Path) -> n.Node:
+    def get_page_ast(self, path: Path) -> n.Root:
```

Comment:
these `n.Root` changes were made so the tests don't fail, since the `n.Node` class does not have the `fileid` option so tests would automatically fail. I'm not 100% sure if converting this to `n.Root` is the correct assumption.

Reply:
Yeah, this is the right move!